salt/utils/win_runas.py

```diff
@@ -49,14 +49,16 @@ def __virtual__():
     return "win_runas"
 
 
-def split_username(username):
-    # TODO: Is there a windows api for this?
-    domain = "."
+def get_username(username):
```

Comment:
I don't think we can rename a function and what it returns without properly deprecating the function. If custom modules are using this function they would would not be able to return the domain or call the split_username function without warning.

Reply:
Nobody is using this but us

Reply:
Reverted